doc/rules/array_notation/no_whitespace_before_comma_in_array.rst

```diff
@@ -77,7 +77,10 @@ The rule is part of the following rule sets:
 
   ``['after_heredoc' => true]``
 
-- `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_
+- `@PhpCsFixer <./../../ruleSets/PhpCsFixer.rst>`_ with config:
+
+  ``['after_heredoc' => true]``
+
 - `@Symfony <./../../ruleSets/Symfony.rst>`_
```

Comment:
side note - if you want to take extra mile, you can check if symfony repo is up to pick this one too

Reply:
Let's land this for `@PhpCsFixer` now. I have no capacity to pursue everything, although I would like it in `@Symfony` ruleset too.